platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -273,8 +273,10 @@ export default function ModeRoute({
       const filters =
         Array.from(query.keys()).reduce(
           (acc: Record<string, string>, val: string) => {
-            if (val !== 'StudyInstanceUIDs') {
-              if (['seriesInstanceUID', 'SeriesInstanceUID'].includes(val)) {
+            const lowerVal = val.toLowerCase();
+            if (lowerVal !== 'studyInstanceuids') {
+              // Note the query to filter the series UIDs is filterSeriesInstanceUID
+              if (lowerVal === 'filterseriesinstanceuid') {
```

Comment:
I thought you didn't change the `SeriesInstnaceUID` filter key, what is `filterseriesinstanceuid`